networkx/algorithms/bipartite/generators.py

```diff
@@ -566,8 +566,6 @@ def gnmk_random_graph(n, m, k, seed=None, directed=False):
     if directed:
         G = nx.DiGraph(G)
     G.name = f"bipartite_gnm_random_graph({n},{m},{k})"
-    if n == 1 or m == 1:
```

Comment:
Returns an empty graph rather than a random star.

Reply:
This will change behavior, no? e.g. n = m = k = 1 (which hits the complete_bipartite_graph branch) you'd get a bipartite graph with two nodes and an edge connecting them, whereas the current behavior gives two nodes with no edges.

I guess the argument is that the current behavior is a bug?

Reply:
I was worried about the case when m = 1, m < k < n. I thought expected behavior should be a random K_(1, k) and n-k isolated vertices. Currently it would just be an empty graph. Also in the case of n = m = k, shouldn't we want a K_2, rather than no isolated nodes?

Reply:
I'm not sure what the behavior should be or what the motivation was for this particular corner-case handling, but you're intuition makes sense to me. Thoughts @MridulS ?

Reply:
> Also in the case of n = m = k, shouldn't we want a K_2, rather than no isolated nodes?

Yeah that does make sense to me.

> I was worried about the case when m = 1, m < k < n. I thought expected behavior should be a random K_(1, k) and n-k isolated vertices.

+1 on this, but again this is just corner-case handling. I'm not sure if we can find proper literature on this?

But anyway if we do go forward with this, we should definitely add this to release notes. Even though it's a "bug fix", it stills changes behavior that may surprise users.